test/e2e/suites/managed/eks_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
Why did you need this change? Asking to understand if we will need it in unmanaged side.